README.rst

```diff
@@ -65,9 +65,11 @@ First of all, you will need to install binaries for Cbc. The version should be 2
 You can install Cbc by either by installing with your system's package manager, by downloading pre-built binaries,
 or by building yourself from source using `coinbrew <https://github.com/coin-or/coinbrew>`_.
 
-1. To install Cbc in Linux, the easiest way is to use a package manager. Install `coinor-libcbc-dev` on Ubuntu/Debian 
-   or `coin-or-Cbc-devel` on Fedora.
-    
+1. To install Cbc in Linux, the easiest way is to use a package manager. Install
+   `coinor-libcbc-dev` on Ubuntu/Debian or `coin-or-Cbc-devel` on Fedora, or the
+   `corresponding package on your distribution
+   <https://doc.sagemath.org/html/en/reference/spkg/cbc.html#equivalent-system-packages>`_.
```

Comment:
Why not point directly to https://repology.org/project/coin-or-cbc/versions? It looks like a more extensive and detailed list, although it doesn't have the convenient command-line invocations. repology.org is a fantastic resource by the way! That list should be added to the Cbc README itself as well. I had no idea about it.

Reply:
I am adding some missing distro information for cbc in https://trac.sagemath.org/ticket/33605

Reply:
>  It looks like a more extensive and detailed list
Note most of the distributions are just rebranded versions of Debian, Arch, Gentoo, ...

Reply:
Ok, makes sense. I would like to add a similar list of distro information to the YAML files for each COIN-OR project and auto-generate similar documentation for their REDAMEs.